ops: jobs: new jobs docs (#784)
inveniosoftware/docs-invenio-rdm: docs/operate/ops/jobs/configure.md

_added in v13.0.0_

## Configuring the Job System

### Config flag
To enable the job system, you need to set the following configuration in your `invenio.cfg` file:

```bash
JOBS_ADMINISTRATION_ENABLED=True
```

### Scheduler

> Invenio Job System uses a custom celery task scheduler which requires you to run a separate - celery beat. You can run the custom beat in a separate container/shell by running:

Reviewer: ```suggestion
The Invenio job system uses a custom celery task scheduler which requires a separate celery beat. You can run the custom beat in a separate container/shell like so:
```

Reviewer: This still needs fixing, not sure if it got lost
